src/site/content/en/fast/codelab-extract-and-inline-critical-css/index.md

```diff
@@ -0,0 +1,198 @@
+---
+layout: codelab
+title: Extract and inline critical CSS with Critical
+authors:
+  - mihajlija
+description: |
+  Learn how use Critical to extract and inline critical CSS, and improve render times.
+glitch: critical-css-starter
+date: 2019-05-08
+related_post: tools-for-extracting-critical-css
+---
+
+Whether you use a UI library or handcraft your styles, shipping a significant amount of CSS delays rendering because the browser must download and parse CSS files before it can show the page.
+
+This responsive ice cream gallery is built with [Bootstrap](https://getbootstrap.com/). UI libraries like Bootstrap speed up the development, but that often comes at the expense of bloated and unnecessary CSS which can slow down your load times. Bootstrap 4 is 187 KB, while [Semantic UI](https://semantic-ui.com/), another UI library, is a whopping 730 KB uncompressed. Even when minified and gzipped, Bootstrap still weighs around 20 KB, well over the 14 KB threshold for the first roundtrip. 
+
+[Critical](https://github.com/addyosmani/critical) is a tool that extracts, minifies and inlines [above-the-fold]() CSS. This helps render styled important content to the user faster while the external CSS file is still being loaded. In this codelab, you’ll learn how to use its npm module. 
```

Comment:
Yeah, was planning to link that to #898 but I wasn't sure if that one is going to end up in /blog or /fast. I'll link to /blog, but the link won't work just yet because the content is on a different branch.